developer-docs-site/docs/guides/building-your-own-wallet.md

```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 
 In order to allow for Aptos' wallet interoperability, the following is required:
 1. Mnemonics - a set of words that can derive account private keys
-2. Dapp API - entry points into the wallet to support access to identity managed by the wallet
+2. dApp API - entry points into the wallet to support access to identity managed by the wallet
```

Comment:
Just a note for now. I am not too keen on spelling it like "dApp" but it's not a major thing. Just something for us to talk about later on. In this PR this is good.

Reply:
Ah, thank you! I was standardizing with the existing:
https://aptos.dev/guides/building-wallet-extension/#step-3-dapp-integration

Please just let me know which way to go forward. Perhaps we can add the desired form to the aforementioned Aptos Words and Phrases list. Thanks, Raj!

Reply:
For now we can just park this issue, I think. No biggie for now.

Reply:
I know you're not keep on `dApp` Raj, but I feel like `dApp` is pretty common in the space